solarforecastarbiter/reports/tests/test_figures.py

```diff
@@ -14,3 +211,39 @@
 def test_abbreviate(arg, expected):
     out = figures.abbreviate(arg)
     assert out == expected
+
+
+def test_timeseries(report_with_raw):
```

Comment:
what happens for this and scatter (and I guess construct cds) with no data?

Reply:
I've added a test for the current behavior. For timeseries, cds creation chokes for no timeseries data (When the ProcessedForecastObservation has `forecast_values` or `observation_values` of type None or str). For Metrics, an empty Tuple returns a cds with only an `index` key that is an empty array.